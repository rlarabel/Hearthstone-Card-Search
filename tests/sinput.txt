Jan'alai, the Dragonhawk
Eldritch Horror
Ice Barrier
Something Wrong
Blood Imp
Nerubian Egg
The Demon Seed
q

