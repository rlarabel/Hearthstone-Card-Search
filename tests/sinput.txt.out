>> Jan'alai, the Dragonhawk
Jan'alai, the Dragonhawk       7
Minion        Mage     Legendary
--------------------------------
[1mBattlecry:[0m If your Hero Power
dealt 8 damage this game,
summon Ragnaros the
Firelord.
--------------------------------
4                              4

>> Eldritch Horror
Eldritch Horror                8
Minion      Neutral       Common
--------------------------------

--------------------------------
6                             10

>> Ice Barrier
Ice Barrier                    3
Spell         Mage        Common
--------------------------------
[1mSecret:[0m When your
hero is attacked,
gain 8 Armor.
--------------------------------
0                              0

>> Something Wrong
./search: 'Something Wrong' not found!
>> Blood Imp
Blood Imp                      1
Minion      Warlock       Common
--------------------------------
  [1mStealth[0m. At the end of your  
turn, give another random
 friendly minion +1 Health.
--------------------------------
0                              1

>> Nerubian Egg
Nerubian Egg                   2
Minion      Neutral         Rare
--------------------------------
[1mDeathrattle:[0m Summon a 4/4 Nerubian.
--------------------------------
0                              2

>> The Demon Seed
The Demon Seed                 1
Spell       Warlock    Legendary
--------------------------------
[1mQuestline:[0m Take 8
damage on your turns.
[1mReward:[0m [1mLifesteal[0m. Deal $3
damage to the enemy hero.
--------------------------------
0                              0

>> q
